cinema/tests/test_movie_session_api.py

```diff
@@ -117,9 +104,7 @@
         response = self.client.get("/api/cinema/movie_sessions/1/")
```

Comment:
The `test_get_movie_session` method uses a hardcoded ID of 1. Ensure that the test database is reset before each test to avoid potential issues with ID assumptions. Alternatively, retrieve the movie session by querying the database to get a valid ID.